### ISSUE TITLE
alternative recommendation

### ISSUE THREAD

**Issue by RustoMCSpit** [linuxmint/mintinstall#383]
say i want discord, flatpak will recommend element or something yoy get the gist. also ability to scan your system and detect apps with foss alternatives and then tell you the pros and cons 

btw, is this the repo for the software manager?

**Comment by corbin-auriti** [comment]
I would say that's out of scope for mintinstall 
Here's a good list if you're interested https://github.com/LinuxCafeFederation/awesome-alternatives
This project does not contain a repository of applications

**Comment by RustoMCSpit** [comment]
> I would say that's out of scope for mintinstall Here's a good list if you're interested https://github.com/LinuxCafeFederation/awesome-alternatives This project does not contain a repository of applications

yes but it could, it could recommenf better front-ends like webcord instead of discord. id say it would be insanely useful for noobs
